src/stores/UserData.ts

```diff
@@ -113,13 +115,36 @@ export const useUserDataStore = defineStore('userDataStore', () => {
         if (!localStorage.getItem('rerollCost')) {
             return {
                 moteCost: 0,
-                tokenCost: 0
+                tokenCost: 0,
+                moteCostStack: 0,
+                tokenCostStack: 0,
             }
         }
 
         return JSON.parse(localStorage.getItem('rerollCost') ?? '{}')
     }
 
+    function getRerollTracker(): RerollTrackerInterface {
+        // Handle Old Local Data
+        if(localStorage.getItem('mod_reroll_tracker')) {
+            const oldRerollCount = localStorage.getItem('mod_reroll_tracker');
+            const newRerollCount = { currentCount: oldRerollCount ? parseInt(oldRerollCount) : 0 }
+            localStorage.removeItem('mod_reroll_tracker');
+            localStorage.setItem('rerollTracker', JSON.stringify(newRerollCount))

```

Comment:
the new reroll format won't save if someone loads the page and doesn't change the reroll tracker without this